agent/consul/server.go

```diff
@@ -141,6 +143,9 @@ type Server struct {
 	// updated
 	reconcileCh chan serf.Member
 
+	// used to track when the server is ready to serve consistent reads
+	readyForConsistentReads uint32
```

Comment:
I'd use `int32` for this since there's no real reason to make it unsigned (as close as atomic gets to a plain old `int`. I'd also put a comment here that atomic instructions are used to access this.